neurons/validator/reward.py

```diff
@@ -349,6 +351,79 @@ def normalize_rewards(self, rewards: torch.FloatTensor) -> torch.FloatTensor:
         return rewards
 
 
+class HumanValidationBotRewardModel(BaseRewardModel):
+    @property
+    def name(self) -> str:
+        return RewardModelType.nsfw.value
+
+    def __init__(self, metagraph, api_url):
+        super().__init__()
+        self.device = "cuda"
+        self.human_voting_bot_scores = torch.zeros((metagraph.n)).to(self.device)
+        self.api_url = api_url
+
+    def get_rewards(self, hotkeys) -> torch.FloatTensor:
+        max_retries = 3
+        backoff = 2
+
+        bt.logging.info("Extracting human votes")
+
+        human_voting_bot_scores = None
```

Comment:
Same here

Reply:
We also have a variable called human_voting_scores though, so please rename to create some distinction. Maybe human_voting_scores_raw/processed?